src/main/java/kostka/moviecatalog/service/MovieEsServiceImpl.java

```diff
@@ -58,6 +59,11 @@ public EsMovie getMovie(final Long movieId) {
         return StreamSupport.stream(movies.spliterator(), false).collect(Collectors.toList());
     }
 
+    @Override
+    public List<EsMovie> get5LatestMovies() {
+        return new ArrayList<>();
```

Comment:
Jestli je to nemenny seznam, dá se nahradit za: `Collections.emptyList()`

Reply:
jasne, díky za hint! :)